plugins/parallel.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Parallel Plugin
+description: The Parallel Plugin
+---
+
+# Parallel Plugin
+
+- [Overview](#overview)
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Laravel](#laravel)
+
+<a name="overview"></a>
+## Overview
+
+Pest is the first testing framework to ship with first-party support for parallel testing. Using this feature,
```

Comment:
@lukeraymonddowning Maybe we can rewrite this differently - something like:

```
By default, Pest execute your tests sequentially within a single process. However, you may greatly reduce the amount of time it takes to run your tests by using the Parallel plugin that allows you run tests simultaneously across multiple processes. 
```

Reply:
Updated